Documentation/General/finalReport/InteractionDesign.tex

```diff
@@ -1,101 +1,88 @@
-We done this with the thinking out loud protocol, so we always knew there they where stuck or did not fully understand something.
-When you do not ask people to think out loud, they would sometimes hide some mistakes, which later on can be very important.
-We gave the users the task to explore the functionalities of the system.
-We got some feedback and found that very valuable.
-Mainly, because in an early stage you know what goes wrong and right. 
-So, anticipating on the results is therefore a lot easier than later on.
-\subsection{GOMS}
-At the end of our project we did a Goals, Operators, Methods, and Selection rules analysis (GOMS).
-We asked them to do some task and watched how quickly they preformed these tasks.
-These where the tasks of the Cognitive Walktrough, only split into multiple sections.
-We wanted to see where sometime went wrong. 
-So, not that the user had some difficulties with finding a chatroom, but that they spent longer on the select a mood page than we would expect.
-These more specific failures gave us more information to work with.
+We have various claims on what we expect of the user.
+In this section we will elaborate the 2 most important claims: user navigation and social interaction.
+
+\subsection{User navigation}
+
+The interface was designed to be minimalistic and intuitive.
+The user should be able to navigate throughout the system without external guidelines or instructions.
+
+We do assume the user is able to understand the ways on how to navigate from the mood selection to a room.
+We have had several iterations of the state transitions to smooth out the user experience.
+Initially, after selecting the moods, the list of rooms was added to the screen.
+After evaluating these transitions with some users, we realized the transition was not clear enough.
+Therefore we changed the behaviour to hide the moods and only show the list of rooms after a mood was selected.
+This resulted in the users understanding the different steps in process of entering a room from a mood.
+
+After changing this, we received additional feedback about the instructions.
+The instructions at the various steps were unclear and could be worded differently.
+We agreed that the instructions could be more fine-grained and decided to place little textboxes on the page.
+These textboxes guide the user, if they require more information.
+Therefore it does not bother users whom already know how to use the system, but will help beginners for which the instructions are aimed.
+
+\subsection{Social interaction}
+
+A room is defined as a collection of users listening to the same song at the same time.
+Additionally we created a chat message system to let users interact with each other.
+
+We claim that users enjoy or prefer to have social interaction with other users.
+For a part of the potential userbase, listening to music is a social activity.
+Good examples of these activities where this is apparant are festivals and pubs.
```

Comment:
apparent

also, "of these activities" -> "where this social element shines" might be a better sentence altogether